packages/schema/lib/schemas/ReadBulkSchema.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const makeSchema = require('../utils/makeSchema');
+
+const BasicDisplaySchema = require('./BasicDisplaySchema');
+const BasicActionOperationSchema = require('./BasicActionOperationSchema');
+const KeySchema = require('./KeySchema');
+
+module.exports = makeSchema(
+  {
+    id: '/ReadBulkSchema',
+    description: 'How will Zapier fetch resources from your application?',
+    type: 'object',
+    required: ['key', 'noun', 'display', 'operation'],
+    examples: [
+      {
+        key: 'recipes',
+        noun: 'Recipes',
+        display: {
+          label: 'Recipes',
+          description: 'A Read that lets Zapier fetch all recipes.',
+        },
+        operation: {
+          perform: '$func$0$f$',
+          sample: {
+            id: 1,
+            firstName: 'Walter',
+            lastName: 'Sobchak',
+            occupation: 'Bowler',
+          },
+        },
+      },
+    ],
+    antiExamples: [
+      {
```

Comment:
anti examples are anything that fails the schema validation. In this case, it would be any object that doesn't have all of `'key', 'noun', 'display', 'operation'` (or one of those isn't the right shape)

Reply:
Ok, I revised. I think that's what we're looking for, but lmk if not.